server/game.py

```diff
@@ -21,6 +21,29 @@ def _configure(max_game_time, agent_dir):
     MAX_GAME_TIME = max_game_time
     AGENT_DIR = agent_dir
 
+def fix_bc_path(path):
```

Comment:
Do you have a sense of why this was not required before / what led the original code here to break?

Reply:
The obvious one is that the bc_model dir saved in the config.pkl file is relative to where it was trained originally, so if the repo is cloned the model dir will not be at that location. 

The reason why this wasn't needed before is that the old agents were all self_play agents (judging from the name), so it probably doesn't need to load a BC model when recreating the trainer. 

Regarding why the bc_model is needed at all, since we are reloading a checkpoint, when recreating the PPOTrainer it needs to recreate the models used at that point. Since we trained the BC model separately, Ray probably doesn't have its weights included in its checkpoint file, so we need to supply that separately.

The reason why we need the trainer to get the agent is just how the code is designed. The trainer does have all the info about its agents, though the downside is it needs the information to create both agents even if we only need 1.

Reply:
I see, that makes sense!